me.amankhanna.opendeck.yaml

```diff
@@ -0,0 +1,35 @@
+id: me.amankhanna.opendeck
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: opendeck
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/ninjadev64/OpenDeck/releases/download/v2.1.2/opendeck_2.1.2_amd64.deb
+        sha256: 567ed33d1eb4220a698751676eded4e9ec34e7ccb7ed30165e0e6fc5047c1442
+        only-arches: [ x86_64 ]
+      - type: file
+        path: me.amankhanna.opendeck.desktop
+      - type: file
+        path: me.amankhanna.opendeck.metainfo.xml
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
+      - install -Dm755 usr/bin/opendeck /app/bin/opendeck
+      - install -Dm644 usr/share/icons/hicolor/512x512/apps/opendeck.png /app/share/icons/hicolor/512x512/apps/me.amankhanna.opendeck.png
+      - install -Dm644 me.amankhanna.opendeck.desktop /app/share/applications/me.amankhanna.opendeck.desktop
```

Comment:
so the .deb comes without a .desktop? It's a bug if it does.

Reply:
See my other comment about the .desktop file.

Reply:
see my other comment too.